cmd/podman/volumes/import.go

```diff
@@ -0,0 +1,75 @@
+package volumes
+
+import (
+	"context"
+	"os"
+
+	"github.com/containers/podman/v3/cmd/podman/common"
+	"github.com/containers/podman/v3/cmd/podman/inspect"
+	"github.com/containers/podman/v3/cmd/podman/parse"
+	"github.com/containers/podman/v3/cmd/podman/registry"
+	"github.com/containers/podman/v3/pkg/domain/entities"
+	"github.com/containers/podman/v3/utils"
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+)
+
+var (
+	importDescription = `Imports contents into a podman volume from specified tarball (.tar, .tar.gz, .tgz, .bzip, .tar.xz, .txz).`
+	importCommand     = &cobra.Command{
+		Annotations:       map[string]string{registry.EngineMode: registry.ABIMode},
+		Use:               "import VOLUME [SOURCE]",
+		Short:             "Import a tarball contents into a podman volume",
+		Long:              importDescription,
+		RunE:              importVol,
+		Args:              cobra.ExactArgs(2),
+		ValidArgsFunction: common.AutocompleteVolumes,
+		Example: `podman volume import my_vol /home/user/import.tar
+  cat ctr.tar | podman import volume my_vol -`,
+	}
+)
+
+func init() {
+	registry.Commands = append(registry.Commands, registry.CliCommand{
+		Command: importCommand,
+		Parent:  volumeCmd,
+	})
+}
+
+func importVol(cmd *cobra.Command, args []string) error {
+	var inspectOpts entities.InspectOptions
+	var tarFile *os.File
+	containerEngine := registry.ContainerEngine()
+	ctx := context.Background()
```

Comment:
just a nit, but no reason for this, just pass `context.Background()` to the `VolumeInspect` function

Reply:
@cdoern sure.